omnibus/config/projects/chef.rb

```diff
@@ -51,15 +51,27 @@
 # Chef Release version pinning
 override :chef, version: "local_source"
 override :ohai, version: "master"
+override :"rb-readline", version: "v0.5.3"
```

Comment:
Looks like @jkeiser is actually changing this in https://github.com/chef/omnibus-software/pull/584